docs/src/content/docs/introduction/overview.mdx

```diff
@@ -67,3 +67,23 @@ When an ASE adopts the Open Payments standard, clients (applications and other p
 - ASEs and clients to create and deploy custom integrations to communicate with one another
 
 Open Payments aims to simplify and democratize payments by providing a standardized, easy-to-integrate solution. This reduces development effort and enhances financial inclusion by making payment solutions more affordable and accessible to all.
+
+## Use cases
+
+Open Payments facilitates various use cases including but not limited to:
+
+### Peer-to-peer payment transfers
+
+As mentioned above, a sender with an account at one Open Payments-enabled ASE can send a payment to a recipient with an account at a different Open Payments-enabled ASE by sending the payment from the sender’s wallet address to the recipient’s wallet address. Thus, Open Payments brings a similar user experience to payments as email brings to person-to-person communications.
+
+### E-commerce platforms
+
+Open Payments can facilitate one-time payments directly between merchants and customers or via third-party marketplaces. A key difference between conventional e-commerce platforms and those enabled by Open Payments is that the payment function won’t be in the control of a sole payment processor or gateway. Refer to the [one-time payment guide](/guides/make-onetime-payment) for more details.
+
+### Recurring payments
+
+In addition to one-time payments that are common on e-commerce marketplaces, Open Payments can also be used to enable subscriptions or payments made at recurring intervals, such as those that are used for Buy Now and Pay Later (BNPL) plans. Refer to the [recurring payment guide](/guides/make-recurring-payments) for more details.
```

Comment:
Marketplaces could support both one-time and recurring payments, so let's leave mention of marketplaces out of the description. Also, I think we can assume the reader will know what a recurring payment is, so we don't need to say its a payment made at a recurring interval. Ideally this section would describe just one use case for recurring payments in more detail, so just (for example) BNPL, or a monthly subscription/donation/bill payment.

Reply:
Added a separate use case for BNPL with a link to the recurring payments guide.